mockwebserver-deprecated/src/test/java/okhttp3/mockwebserver/MockWebServerTest.java

```diff
@@ -61,15 +62,20 @@
 import static org.junit.Assume.assumeFalse;
 
 @SuppressWarnings({"ArraysAsListWithZeroOrOneArgument", "deprecation"})
+@Timeout(30)
 public final class MockWebServerTest {
-  @Rule public PlatformRule platform = new PlatformRule();
+  @RegisterExtension public PlatformRule platform = new PlatformRule();
 
-  @Rule public final MockWebServer server = new MockWebServer();
```

Comment:
Not clear to me this test should be migrated because it's currently a regression test.